deploys/demo.syncfor.science/config/vendors/epic.yml

```diff
@@ -13,6 +13,7 @@ auth:
   sign_in_steps:
     - element: "#Login"
       send_keys: "fhirjason"
+      wait: 5
```

Comment:
I'm still not sure about this wait: would it work to have a longer implicit wait (so we don't just spin our fingers waiting 5s in the event that the field appears in 2s? Also, hard-coded forced waits are pretty fragile). Also webdriver [docs](http://www.seleniumhq.org/docs/04_webdriver_advanced.jsp) warn "WARNING: Do not mix implicit and explicit waits. Doing so can cause unpredictable wait times."